news/2023/2023-02-09-Skin-of-the-Year-2022.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Skin of the Year 2022: Voting Open"
+date: 2023-02-09 09:00:00 +0000
+---
+
+Do you think you got what it takes to select the BEST skins of 2022? Let us know by casting your vote!
+
+![](/wiki/shared/news/2023-02-01-Skin-of-the-Year-2022/SOTY-2022-banner.jpg)
```

Comment:
The folder is dated to the incorrect date

Reply:
fixed in https://github.com/ppy/osu-wiki/pull/8773/commits/9a7d28e56aba37e39ffce210c2e5f556ef7a2f4d